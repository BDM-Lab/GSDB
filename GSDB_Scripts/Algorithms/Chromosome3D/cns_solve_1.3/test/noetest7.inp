remarks  test NOE restraints facility
remarks  

topology @CNS_TOPPAR:protein.top end
parameter @CNS_TOPPAR:protein.param end

segment
   name=mol1
   molecule  name=lys number=1 end
end
segment
   name=mol2
   molecule name=lys number=1 end
end

delete selection=( not ( name nz or name hz* or name ce )) end

coor
remarks test
ATOM      1  CE  LYS     1      -0.484   1.367   0.000  1.00  0.00      MOL1
ATOM      2  NZ  LYS     1       0.000   0.000   0.000  1.00  0.00      MOL1
ATOM      3  HZ1 LYS     1       1.040   0.000   0.000  1.00  0.00      MOL1
ATOM      4  HZ2 LYS     1      -0.346  -0.491   0.849  1.00  0.00      MOL1
ATOM      5  HZ3 LYS     1      -0.346  -0.491  -0.849  1.00  0.00      MOL1
ATOM      6  CE  LYS     1      -0.484   1.367   0.000  1.00  0.00      MOL2
ATOM      7  NZ  LYS     1       0.000   0.000   0.000  1.00  0.00      MOL2
ATOM      8  HZ1 LYS     1       1.040   0.000   0.000  1.00  0.00      MOL2
ATOM      9  HZ2 LYS     1      -0.346  -0.491   0.849  1.00  0.00      MOL2
ATOM     10  HZ3 LYS     1      -0.346  -0.491  -0.849  1.00  0.00      MOL2
END

coor rotate   angle=180.0   selection=( segid mol2 )  vector=( 0.0 0.0 1.0) end

coor translate vector=( 1.0 0.0 0.0 ) dist=5.0  selection=( segid mol2 ) end


flags  exclude * include noe   end

noe
  reset
  nrestraints=100
  temperature=300.0
  class one
end



{* test NOE PREDict facility *}
{* ========================= *}
noe


   assign ( atom mol1 1 hz* ) ( atom mol2 1 hz*)  4.0  0.08 0.17

   print thres=-1
   outden
end

coor translate vector=( 0.5 0.0 0.0 ) dist=3.0  selection=( segid mol2 ) end

noe 
   den init
   outden
end

coor translate vector=( 1.0 0.0 0.0 ) dist=3.0  selection=( segid mol2 ) end


noe
   print thres=-1

   den update gamma=0.4 kappa=0.3
   
   print thres=-1

   den update gamma=0.4 kappa=0.3
   
   print thres=-1

   den update gamma=0.4 kappa=0.3
   
   print thres=-1
   
   den update gamma=1 kappa=1
   
   print thres=-1
   
   outden
   
end

stop


   
