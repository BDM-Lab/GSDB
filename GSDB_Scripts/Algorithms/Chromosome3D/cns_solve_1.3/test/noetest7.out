          ============================================================
          |                                                          |
          |            Crystallography & NMR System (CNS)            |
          |                         CNSsolve                         |
          |                                                          |
          ============================================================
           Version: 1.3
           Status: General release
          ============================================================
           Written by: A.T.Brunger, P.D.Adams, G.M.Clore, W.L.DeLano,
                       P.Gros, R.W.Grosse-Kunstleve,J.-S.Jiang,J.M.Krahn,
                       J.Kuszewski, M.Nilges, N.S.Pannu, R.J.Read,
                       L.M.Rice, G.F.Schroeder, T.Simonson, G.L.Warren.
           Copyright (c) 1997-2010 Yale University
          ============================================================
           Running on machine: lewis4-r710-login-node223 (x86_64/Linux,64-bit)
                               with 16 threads
           Program started by: oeow39
           Program started at: 16:08:04 on 05-Jul-2018
          ============================================================

 FFT3C: Using FFTPACK4.1

 CNSsolve>remarks  test NOE restraints facility 
 CNSsolve>remarks 
 CNSsolve> 
 CNSsolve>topology @CNS_TOPPAR:protein.top end 
 ASSFIL: file protein.top opened.
 RTFRDR>remarks file toppar/protein.top 
 RTFRDR>remarks   protein topology with polar hydrogens 
 RTFRDR> 
 RTFRDR>! 
 RTFRDR>! Please cite the following reference when using these parameters: 
 RTFRDR>! Engh, R.A. and  Huber, R. (1991). Accurate Bond and 
 RTFRDR>!  Angle Parameters for X-ray Protein-Structure Refinement, 
 RTFRDR>!  Acta Cryst. A47, 392-400. 
 RTFRDR>! 
 RTFRDR>! 
 RTFRDR> 
 RTFRDR>set message ? end eval ($old_message=$result) set echo ? end eval ($old_echo=$result) 
 MESSage=NORM
 EVALUATE: symbol $OLD_MESSAGE set to "NORM" (string)
 ECHO=TRUE {ON}
 EVALUATE: symbol $OLD_ECHO set to TRUE (logical)
 RTFRDR>set message=off echo=off end 
 RTFRDR> 
 RTFRDR> 
 RTFRDR> end 
 CNSsolve>parameter @CNS_TOPPAR:protein.param end 
 ASSFIL: file protein.param opened.
 PARRDR>remarks file toppar/protein.param 
 PARRDR>remarks   protein parameters with Lennard-Jones van der Waals parameters 
 PARRDR>remarks   for crystallographic structure determination 
 PARRDR> 
 PARRDR>! 
 PARRDR>! Please cite the following reference when using these parameters: 
 PARRDR>! Engh, R.A. and  Huber, R. (1991). Accurate Bond and 
 PARRDR>!  Angle Parameters for X-ray Protein-Structure Refinement, 
 PARRDR>!  Acta Cryst. A47, 392-400. 
 PARRDR>! 
 PARRDR>! 
 PARRDR> 
 PARRDR>set message ? end eval ($old_message=$result) set echo ? end eval ($old_echo=$result) 
 MESSage=NORM
 EVALUATE: symbol $OLD_MESSAGE set to "NORM" (string)
 ECHO=TRUE {ON}
 EVALUATE: symbol $OLD_ECHO set to TRUE (logical)
 PARRDR>set message=off echo=off end 
 PARRDR> 
 PARRDR> end 
 CNSsolve> 
 CNSsolve>segment 
 SEGMENT>   name=mol1 
 SEGMENT>   molecule  name=lys number=1 end 
 SEGMENT>end 
 Status of internal molecular topology database:
 -> NATOM=         13(MAXA=     1000000)  NBOND=         12(MAXB=     1000000)
 -> NTHETA=        15(MAXT=     2000000)  NGRP=           1(MAXGRP=   1000000)
 -> NPHI=           7(MAXP=     2000000)  NIMPHI=         1(MAXIMP=   1000000)
 -> NNB=            0(MAXNB=    1000000) 
 CNSsolve>segment 
 SEGMENT>   name=mol2 
 SEGMENT>   molecule name=lys number=1 end 
 SEGMENT>end 
 Status of internal molecular topology database:
 -> NATOM=         26(MAXA=     1000000)  NBOND=         24(MAXB=     1000000)
 -> NTHETA=        30(MAXT=     2000000)  NGRP=           2(MAXGRP=   1000000)
 -> NPHI=          14(MAXP=     2000000)  NIMPHI=         2(MAXIMP=   1000000)
 -> NNB=            0(MAXNB=    1000000) 
 CNSsolve> 
 CNSsolve>delete selection=( not ( name nz or name hz* or name ce )) end 
 SELRPN:     16 atoms have been selected out of     26
 Status of internal molecular topology database:
 -> NATOM=         10(MAXA=     1000000)  NBOND=          8(MAXB=     1000000)
 -> NTHETA=        12(MAXT=     2000000)  NGRP=           2(MAXGRP=   1000000)
 -> NPHI=           0(MAXP=     2000000)  NIMPHI=         0(MAXIMP=   1000000)
 -> NNB=            0(MAXNB=    1000000) 
 CNSsolve> 
 CNSsolve>coor 
 COOR>remarks test 
 COOR>ATOM      1  CE  LYS     1      -0.484   1.367   0.000  1.00  0.00      MOL1 
 COOR>ATOM      2  NZ  LYS     1       0.000   0.000   0.000  1.00  0.00      MOL1 
 CNSsolve> 
 CNSsolve>coor rotate   angle=180.0   selection=( segid mol2 )  vector=( 0.0 0.0 1.0) end 
 SELRPN:      5 atoms have been selected out of     10
 COOR: using atom subset.

 Rotation matrix =
   -1.000000    0.000000    0.000000
    0.000000   -1.000000    0.000000
    0.000000    0.000000    1.000000
 Corresp. Eulerian angles (theta1,theta2,theta3)   180.0000    0.0000    0.0000
 Corresp. spherical polar angles (psi,phi,kappa)    90.0000   90.0000  180.0000
 Corresp. rotation angle   180.0000 about axis       0.0000    0.0000   -1.0000
 Corresp. quaternions     0.0000    0.0000    0.0000    1.0000

 COOR: rotation center =    0.000000    0.000000    0.000000
 COOR: selected coordinates rotated
 CNSsolve> 
 CNSsolve>coor translate vector=( 1.0 0.0 0.0 ) dist=5.0  selection=( segid mol2 ) end 
 SELRPN:      5 atoms have been selected out of     10
 COOR: using atom subset.
 COOR: translation vector =(     5.000000    0.000000    0.000000 )
 COOR: selected coordinates translated
 CNSsolve> 
 CNSsolve> 
 CNSsolve>flags  exclude * include noe   end 
 CNSsolve> 
 CNSsolve>noe 
 NOE>  reset 
 NOE>  nrestraints=100 
 NOE: allocating space for     100 restraints.
 NOE>  temperature=300.0 
 NOE>  class one 
 NOE>end 
 CNSsolve> 
 CNSsolve> 
 CNSsolve> 
 CNSsolve>{* test NOE PREDict facility *} 
 CNSsolve>{* ========================= *} 
 CNSsolve>noe 
 NOE> 
 NOE> 
 NOE>   assign ( atom mol1 1 hz* ) ( atom mol2 1 hz*)  4.0  0.08 0.17 
 SELRPN:      3 atoms have been selected out of     10
 SELRPN:      3 atoms have been selected out of     10
 NOE> 
 NOE>   print thres=-1 

 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 +++++++++++++++++++++++++++++++ CLASS ONE  +++++++++++++++++++++++++++++++++++
 for this class: SCALe=   1.000 AVERage=R-6    POTEntial=biharmonic 
                 TEMPerature= 300.000
 
 ========== restraint     1 ==========
 set-i-atoms
          MOL1 1    LYS  HZ1 
          MOL1 1    LYS  HZ2 
          MOL1 1    LYS  HZ3 
 set-j-atoms
          MOL2 1    LYS  HZ1 
          MOL2 1    LYS  HZ2 
          MOL2 1    LYS  HZ3 
 R<average>=   3.985 NOE= 4.00 (- 0.08/+ 0.17) Delta=   0.015  E(NOE)=   0.007

 NOEPRI: RMS diff. =   0.015,  #(violat.>-1.0)=     1 of      1 NOEs
 NOEPRI: RMS diff. class ONE  =   0.015,  #(viol.>-1.0)=     1 of      1 NOEs
 NOE>   outden 
 ASSIgn ( KNOWn and ATOM "MOL1" "1   " "HZ1 " ) ( KNOWn and ATOM "MOL2" "1   " "HZ1 " )  4.0000  0.080  0.170
 NOE>end 
 CNSsolve> 
 CNSsolve>coor translate vector=( 0.5 0.0 0.0 ) dist=3.0  selection=( segid mol2 ) end 
 SELRPN:      5 atoms have been selected out of     10
 COOR: using atom subset.
 COOR: translation vector =(     3.000000    0.000000    0.000000 )
 COOR: selected coordinates translated
 CNSsolve> 
 CNSsolve>noe 
 NOE>   den init 
 NOE>   outden 
 ASSIgn ( KNOWn and ATOM "MOL1" "1   " "HZ1 " ) ( KNOWn and ATOM "MOL2" "1   " "HZ1 " )  4.0000  0.080  0.170
 NOE>end 
 CNSsolve> 
 CNSsolve>coor translate vector=( 1.0 0.0 0.0 ) dist=3.0  selection=( segid mol2 ) end 
 SELRPN:      5 atoms have been selected out of     10
 COOR: using atom subset.
 COOR: translation vector =(     3.000000    0.000000    0.000000 )
 COOR: selected coordinates translated
 CNSsolve> 
 CNSsolve> 
 CNSsolve>noe 
 NOE>   print thres=-1 

 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 +++++++++++++++++++++++++++++++ CLASS ONE  +++++++++++++++++++++++++++++++++++
 for this class: SCALe=   1.000 AVERage=R-6    POTEntial=biharmonic 
                 TEMPerature= 300.000
 
 ========== restraint     1 ==========
 set-i-atoms
          MOL1 1    LYS  HZ1 
          MOL1 1    LYS  HZ2 
          MOL1 1    LYS  HZ3 
 set-j-atoms
          MOL2 1    LYS  HZ1 
          MOL2 1    LYS  HZ2 
          MOL2 1    LYS  HZ3 
 R<average>=  10.499 NOE= 4.00 (- 0.08/+ 0.17) Delta=  -6.499  E(NOE)= 435.586

 NOEPRI: RMS diff. =   6.499,  #(violat.>-1.0)=     1 of      1 NOEs
 NOEPRI: RMS diff. class ONE  =   6.499,  #(viol.>-1.0)=     1 of      1 NOEs
 NOE> 
 NOE>   den update gamma=0.4 kappa=0.3 
 NOE> 
 NOE>   print thres=-1 

 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 +++++++++++++++++++++++++++++++ CLASS ONE  +++++++++++++++++++++++++++++++++++
 for this class: SCALe=   1.000 AVERage=R-6    POTEntial=biharmonic 
                 TEMPerature= 300.000
 
 ========== restraint     1 ==========
 set-i-atoms
          MOL1 1    LYS  HZ1 
          MOL1 1    LYS  HZ2 
          MOL1 1    LYS  HZ3 
 set-j-atoms
          MOL2 1    LYS  HZ1 
          MOL2 1    LYS  HZ2 
          MOL2 1    LYS  HZ3 
 R<average>=  10.499 NOE= 5.38 (- 0.08/+ 0.17) Delta=  -5.114  E(NOE)= 269.743

 NOEPRI: RMS diff. =   5.114,  #(violat.>-1.0)=     1 of      1 NOEs
 NOEPRI: RMS diff. class ONE  =   5.114,  #(viol.>-1.0)=     1 of      1 NOEs
 NOE> 
 NOE>   den update gamma=0.4 kappa=0.3 
 NOE> 
 NOE>   print thres=-1 

 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 +++++++++++++++++++++++++++++++ CLASS ONE  +++++++++++++++++++++++++++++++++++
 for this class: SCALe=   1.000 AVERage=R-6    POTEntial=biharmonic 
                 TEMPerature= 300.000
 
 ========== restraint     1 ==========
 set-i-atoms
          MOL1 1    LYS  HZ1 
          MOL1 1    LYS  HZ2 
          MOL1 1    LYS  HZ3 
 set-j-atoms
          MOL2 1    LYS  HZ1 
          MOL2 1    LYS  HZ2 
          MOL2 1    LYS  HZ3 
 R<average>=  10.499 NOE= 6.35 (- 0.08/+ 0.17) Delta=  -4.145  E(NOE)= 177.185

 NOEPRI: RMS diff. =   4.145,  #(violat.>-1.0)=     1 of      1 NOEs
 NOEPRI: RMS diff. class ONE  =   4.145,  #(viol.>-1.0)=     1 of      1 NOEs
 NOE> 
 NOE>   den update gamma=0.4 kappa=0.3 
 NOE> 
 NOE>   print thres=-1 

 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 +++++++++++++++++++++++++++++++ CLASS ONE  +++++++++++++++++++++++++++++++++++
 for this class: SCALe=   1.000 AVERage=R-6    POTEntial=biharmonic 
                 TEMPerature= 300.000
 
 ========== restraint     1 ==========
 set-i-atoms
          MOL1 1    LYS  HZ1 
          MOL1 1    LYS  HZ2 
          MOL1 1    LYS  HZ3 
 set-j-atoms
          MOL2 1    LYS  HZ1 
          MOL2 1    LYS  HZ2 
          MOL2 1    LYS  HZ3 
 R<average>=  10.499 NOE= 7.03 (- 0.08/+ 0.17) Delta=  -3.466  E(NOE)= 123.924

 NOEPRI: RMS diff. =   3.466,  #(violat.>-1.0)=     1 of      1 NOEs
 NOEPRI: RMS diff. class ONE  =   3.466,  #(viol.>-1.0)=     1 of      1 NOEs
 NOE> 
 NOE>   den update gamma=1 kappa=1 
 NOE> 
 NOE>   print thres=-1 

 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 +++++++++++++++++++++++++++++++ CLASS ONE  +++++++++++++++++++++++++++++++++++
 for this class: SCALe=   1.000 AVERage=R-6    POTEntial=biharmonic 
                 TEMPerature= 300.000
 
 ========== restraint     1 ==========
 set-i-atoms
          MOL1 1    LYS  HZ1 
          MOL1 1    LYS  HZ2 
          MOL1 1    LYS  HZ3 
 set-j-atoms
          MOL2 1    LYS  HZ1 
          MOL2 1    LYS  HZ2 
          MOL2 1    LYS  HZ3 
 R<average>=  10.499 NOE=10.50 (- 0.08/+ 0.17) Delta=   0.000  E(NOE)=   0.000

 NOEPRI: RMS diff. =   0.000,  #(violat.>-1.0)=     1 of      1 NOEs
 NOEPRI: RMS diff. class ONE  =   0.000,  #(viol.>-1.0)=     1 of      1 NOEs
 NOE> 
 NOE>   outden 
 ASSIgn ( KNOWn and ATOM "MOL1" "1   " "HZ1 " ) ( KNOWn and ATOM "MOL2" "1   " "HZ1 " ) 10.4986  0.080  0.170
 NOE> 
 NOE>end 
 CNSsolve> 
 CNSsolve>stop 
          ============================================================
           Maximum dynamic memory allocation:     8251608 bytes
           Maximum dynamic memory overhead:           544 bytes
           Program started at: 16:08:04 on 05-Jul-2018
           Program stopped at: 16:08:04 on 05-Jul-2018
           CPU time used:       0.0178 seconds
          ============================================================
